install/kubernetes/helm/istio/charts/certmanager/templates/_helpers.tpl

```diff
@@ -0,0 +1,35 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "certmanager.name" -}}
```

Comment:
I'll ask again - what is this doing ?

Reply:
I mean - when would the name be different than the default ?

Reply:
It makes the same thing as grafana or prometheus helpers

Reply:
Yes, they should be fixed as well. We can do that later I guess, but more people will follow this pattern.
Many others charts don't do this.